tests/src/Functional/PersonContentRenderTest.php

```diff
@@ -143,13 +144,30 @@ public function testPersonPage(): void {
 
     $document_group_title = $content->filter('h3.fs-4');
     $this->assertEquals('Curriculum Vitae', $document_group_title->text());
-    $files = $content->filter('.bcl-file');
-    $this->assertEquals(
-      'Person document test',
-      $files->filter('p')->text()
-    );
+    $files = $content->filter('.bcl-file-container');
 
+    $document_file = File::create([
+      'uri' => $this->getTestFiles('text')[0]->uri,
+    ]);
+    $document_file->save();
+    $absolute_file_url = $document_file->createFileUrl(FALSE);
```

Comment:
absolute_file_url is not needed, you can call $document_file->createFileUrl(FALSE); directly in $expected_document

Reply:
done. thanks